build/dependencies.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
-    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <CoreFxVersion>4.4.0-*</CoreFxVersion>
```

Comment:
Is this a required version bump? I didn't make the same change in aspnet/Localization and see @JunTaoLuo left https://github.com/aspnet/HttpAbstractions/pull/840/files#diff-30dcc9789f8ea3c7632e4f054e60c072R4 alone in his similar PR.